js/main.js

```diff
@@ -22,4 +41,60 @@ function saveEntryHandle(event) {
   data.nextEntryId++;
   $imgEntry.setAttribute('src', 'images/placeholder-image-square.jpg');
   $formElement.reset();
+  entriesPage(event);
```

Comment:
`entriesPage` doesn't have a parameter, so you don't need to pass an argument to the function.